manage.py

```diff
@@ -5,6 +5,7 @@
 
 if __name__ == '__main__':
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings.local')
+    os.environ.setdefault('DJANGO_DOT_ENV_FILE', '.env')
```

Comment:
I will remove this, as on production we inject the environments from kubernetes and I bet this will conflict